docker/production.yml

```diff
@@ -49,6 +50,7 @@ services:
       - API_URL
       - WEB_URL
       - SEARCH_URL
```

Comment:
@humphd 
same here for: `  'AUTH_URL'` and `'USERS_URL'` ?
reference: https://github.com/Seneca-CDOT/telescope/pull/2148#discussion_r614091986

Reply:
Yes, we need `AUTH_URL` in here, and if we're going to include `USERS_URL` above, we need it here too.

Reply:
Okay.  
Tks. 

Done.